### PR TITLE
Puppy Ian. Awwww (And a puppy crate)

### PR DESCRIPTION
**Pull request by VasilisThePikachu**
## About the PR

This PR will add a puppy ian and add it to one of the random ian spawners for hop. Awww

## Media

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

image
image

**Changelog**

:cl:
- add: Added a Puppy Ian, Awww.
- add: Puppy Ian can be randomly chosen to be hop's pet along with the other corgis.
- add: Added a puppy corgi crate, it contains a single puppy for hard times. It can be order by cargo.

### CODE REVIEW

File: Resources/Prototypes/Entities/Mobs/NPCs/pets.yml [hunk 1]
```diff
@@ -83,6 +83,21 @@
...
+  id: MobCorgiIanPup
+  description: Favourite puppy corgi. Awww.
+  components:
+  - type: Grammar
+    attributes:
+      proper: true
+      gender: male
+  - type: Butcherable # A puppy? You monster...
+    spawned:
+    - id: FoodMeatCorgi
```
Comment (deltanedas) [inline Resources/Prototypes/Entities/Mobs/NPCs/pets.yml:98]: no corgi hide :trollface:

whats better than wearing ian suit? puppy ian suit

Comment (VasilisThePikachu) [reply Resources/Prototypes/Entities/Mobs/NPCs/pets.yml:98]: You don't join the discord now this? You monster

**Comment by notquitehadouken** [comment]
now add an objective for stealing his hide :trollface:

**Comment by deltanedas** [comment]
corgi meat objective (removed) except you redtext if you make ian suit (bad)

**Comment by VasilisThePikachu** [comment]
Based Emisse

**Pull request merged**